<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>realizar_venta_window</class>
 <widget class="QMainWindow" name="realizar_venta_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1090</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Realizar Venta</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../assets/resources.qrc">
    <normaloff>:/images/001-cart.png</normaloff>:/images/001-cart.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="venta_tableWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>1031</width>
      <height>351</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="gridStyle">
     <enum>Qt::NoPen</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>550</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Total:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="total_label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>550</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;$0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>1071</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>25</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Agregar por:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="buscar_producto_por_comboBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>21</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="termino_producto_lineEdit">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>20</y>
       <width>511</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Termino busqueda</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="agregar_producto_pushButton">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>20</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Agregar producto</string>
     </property>
     <property name="icon">
      <iconset resource="../assets/resources.qrc">
       <normaloff>:/images/020-cart.png</normaloff>:/images/020-cart.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>1031</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Realizar Venta&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="finalizar_venta_pushButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>620</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Finalizar Venta</string>
    </property>
    <property name="icon">
     <iconset resource="../assets/resources.qrc">
      <normaloff>:/images/021-cart.png</normaloff>:/images/021-cart.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelar_pushButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>620</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
    <property name="icon">
     <iconset resource="../assets/resources.qrc">
      <normaloff>:/images/028-cart.png</normaloff>:/images/028-cart.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="disminuir_pushButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>540</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Disminuir</string>
    </property>
    <property name="icon">
     <iconset resource="../assets/resources.qrc">
      <normaloff>:/images/014-bookmark-5.png</normaloff>:/images/014-bookmark-5.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="aumentar_pushButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>540</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Aumentar</string>
    </property>
    <property name="icon">
     <iconset resource="../assets/resources.qrc">
      <normaloff>:/images/019-bookmark.png</normaloff>:/images/019-bookmark.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="eliminar_producto_pushButton">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>540</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Eliminar producto</string>
    </property>
    <property name="icon">
     <iconset resource="../assets/resources.qrc">
      <normaloff>:/images/027-cart.png</normaloff>:/images/027-cart.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1090</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../assets/resources.qrc"/>
 </resources>
 <connections/>
</ui>
